### ISSUE TITLE
example doesn't work

### ISSUE THREAD

**Issue by rohrdr** [qubvel/segmentation_models.pytorch#38]
I wasn't able to get the example with the cars segmentation to run.
Changing the code in unet/decoder.py for the class DecoderBlock worked for me:

    def forward(self, x):
        x, skip = x
        if skip is not None:
            x = F.interpolate(x, size=(skip.shape[-2], skip.shape[-1]), mode='nearest')
            x = torch.cat([x, skip], dim=1)
        else:
            x = F.interpolate(x, scale_factor=2, mode='nearest')
        x = self.block(x)
        return x
